amr-wind/physics/multiphase/DamBreak.H

```diff
@@ -0,0 +1,47 @@
+#ifndef DamBreak_H
+#define DamBreak_H
+
+#include "amr-wind/core/Physics.H"
+#include "amr-wind/physics/multiphase/DamBreakFieldInit.H"
+#include "amr-wind/core/Field.H"
+
+/** Multiphase Dam break physics
+ *  \ingroup mulitphase_physics
+ *
+ */
+
+namespace amr_wind {
+
+class DamBreak : public Physics::Register<DamBreak>
```

Comment:
Do you anticipate DamBreak doing anything beyond just field initialization? If not, then you don't need a separate FieldInit class, just have DamBreak do everything. In ABL we broke it out into parts because it had several other pieces it was orchestrating. 

Also recommend adding a doxygen style comment here explaining this problem and adding relevant references.

Reply:
Thanks, I was wondering if I should do this. Mike also suggesting having just DamBreak.H/cpp and dropping the initialization class. I will add comments etc.